### PR TITLE
chore: update English language capitalizations and format

### PR DESCRIPTION
**Pull request by PalmDevs**

### CODE REVIEW

File: assets/i18n/en.json [hunk 1]
```diff
@@ -34,22 +34,22 @@
         "changelogLabel": "Changelog"
     },
     "latestCommitCard": {
-        "loadingLabel": "Loading",
+        "loadingLabel": "Loading...",
         "timeagoLabel": "{time} ago",
         "patcherLabel": "Patcher: ",
         "managerLabel": "Manager: ",
-        "updateButton": "Update Manager"
+        "updateButton": "Update"
```
Comment (ponces) [inline assets/i18n/en.json:41]: removing "Manager" will make this button ambiguous as to what it does for the user. I don't like this change

File: assets/i18n/en.json [hunk 2]
```diff
@@ -102,22 +102,22 @@
         "teamSectionTitle": "Team",
         "infoSectionTitle": "Info",
         "advancedSectionTitle": "Advanced",
-        "darkThemeLabel": "Dark Mode",
-        "darkThemeHint": "Welcome to the Dark Side",
+        "darkThemeLabel": "Dark mode",
```
Comment (ponces) [inline assets/i18n/en.json:105]: This should be capitalized

File: assets/i18n/en.json [hunk 3]
```diff
@@ -102,22 +102,22 @@
...
+        "darkThemeHint": "Welcome to the dark side",
```
Comment (ponces) [inline assets/i18n/en.json:106]: This should be capitalized

File: assets/i18n/en.json [hunk 4]
```diff
@@ -102,22 +102,22 @@
...
         "languageLabel": "Language",
         "englishOption": "English",
         "frenchOption": "French",
         "sourcesLabel": "Sources",
         "sourcesLabelHint": "Configure your custom sources",
-        "orgPatchesLabel" : "Patches Org",
-        "sourcesPatchesLabel" : "Patches Source",
-        "orgIntegrationsLabel": "Integrations Org",
-        "sourcesIntegrationsLabel": "Integrations Source",
+        "orgPatchesLabel" : "Patches organization",
```
Comment (ponces) [inline assets/i18n/en.json:114]: This should be capitalized

File: assets/i18n/en.json [hunk 5]
```diff
@@ -102,22 +102,22 @@
...
+        "sourcesPatchesLabel" : "Patches source",
```
Comment (ponces) [inline assets/i18n/en.json:115]: This should be capitalized

File: assets/i18n/en.json [hunk 6]
```diff
@@ -102,22 +102,22 @@
...
+        "orgIntegrationsLabel": "Integrations organization",
```
Comment (ponces) [inline assets/i18n/en.json:116]: This should be capitalized

File: assets/i18n/en.json [hunk 7]
```diff
@@ -102,22 +102,22 @@
...
+        "sourcesIntegrationsLabel": "Integrations source",
```
Comment (ponces) [inline assets/i18n/en.json:117]: This should be capitalized

**Comment by Aunali321** [comment]
Yup agree with you. Please make the changes @PalmDevs .

**Comment by PalmDevs** [comment]
Checked from Google apps and text in those contexts do not need the capitalization. But sure, I would revert for you

**Comment by PalmDevs** [comment]
I do half agree on the `xxx organization` part though

**Comment by PalmDevs** [comment]
@Aunali321, please review

**Comment by Aunali321** [comment]
In latest commit card, it should be Update Manager not Update.

**Comment by PalmDevs** [comment]
Check latest commit, it has Update Manager

**Comment by PalmDevs** [comment]
@ponces May I ask why `Mode` in `Dark Mode` needs to be capitalized?

**Comment by PalmDevs** [comment]
> Check latest commit, it has Update Manager 

@Aunali321 Can you review again, and possibly merge?

**Comment by PalmDevs** [comment]
@Aunali321 REVIEW LAST TIME PLEASE THX

**Comment by Aunali321** [comment]
Looks good

**Comment by Aunali321** [comment]
Thanks!

**Comment by PalmDevs** [comment]
COSSALE SHALL MERGE.

**Pull request merged**